lib/ansible/utils/module_docs_fragments/rackspace.py

```diff
@@ -74,6 +74,11 @@ class ModuleDocFragment(object):
       - The URI of the authentication service.
     default: https://identity.api.rackspacecloud.com/v2.0/
     version_added: 1.5
+  auth_token:
+    description:
+      - A pre-generated authentication token (requires I(tenant_name) or C(OS_PROJECT_NAME))
+        see U(https://developer.rackspace.com/docs/cloud-identity/v2/api-reference/token-operations/).
+    version_added: 2.6.0
```

Comment:
Update this to `2.8` since that will be the upcoming release this could be included in.